assets/overlays/aws-ebs/aws_ebs.go

```diff
@@ -70,3 +104,315 @@ func GetAWSEBSGeneratorConfig() *generator.CSIDriverGeneratorConfig {
 		},
 	}
 }
+
+func GetAWSEBSOperatorConfig() *config.OperatorConfig {
+	return &config.OperatorConfig{
+		CSIDriverName:                   opv1.AWSEBSCSIDriver,
+		UserAgent:                       "aws-ebs-csi-driver-operator",
+		AssetReader:                     assets.ReadFile,
+		AssetDir:                        generatedAssetBase,
+		OperatorControllerConfigBuilder: GetAWSEBSOperatorControllerConfig,
+	}
+}
+
+func GetAWSEBSOperatorControllerConfig(flavour generator.ClusterFlavour, c *clients.Clients) *config.OperatorControllerConfig {
+	cfg := defaults.NewDefaultOperatorConfig(flavour, c, "AWSEBS")
+
+	// Hooks to run on all clusters
+	cfg.AddDeploymentHookBuilders(c,
+		withAWSRegion,
+		withCustomTags,
+		withCustomEndPoint,
+		withCABundleDeploymentHook)
+	cfg.AddDaemonSetHookBuilders(c, withCABundleDaemonSetHook)
+	cfg.AddStorageClassHookBuilders(c, withKMSKeyHook)
+
+	if flavour == generator.FlavourStandalone {
+		// Standalone-only hooks
+		cfg.AddDeploymentHookBuilders(c, getCustomAWSCABundleBuilder(cloudConfigName))
+	} else {
+		// HyperShift only hooks
+		cfg.AddDeploymentHookBuilders(c, getCustomAWSCABundleBuilder("user-ca-bundle"))
+	}
+	cfg.DeploymentWatchedSecretNames = append(cfg.DeploymentWatchedSecretNames, cloudCredSecretName, metricsCertSecretName)
+
+	// Controllers to run on all clusters
+	ctrl, err := newCustomAWSBundleSyncer(c)
```

Comment:
Does this needs to happen in hypershift clusters too?

Reply:
Good catch, this is standalone only. Fixed